arbnode/node.go

```diff
@@ -1288,6 +1289,14 @@ func SetUpDataAvailability(
 		if err != nil {
 			return nil, nil, err
 		}
+		if config.RedisCacheConfig.SyncFromStorageServices {
+			iterableStorageService := das.NewIterableStorageService(das.ConvertStorageServiceToIterationCompatibleStorageService(cache))
```

Comment:
My assumption was that redis will not have an expiration time, but makes sense to be sure, so setting expiration as 0(never expire) for putKeyValue call so the index never expires.

This will not be an issue in future since we plan to have index cleanup process in place.